rfcs/single_test.md

```diff
@@ -0,0 +1,301 @@
+- [/workers/constructors/SharedWorker/URLMismatchError.htm](https://wpt.fyi/results/workers/constructors/SharedWorker/URLMismatchError.htm?run_id=310440006&run_id=297430002&run_id=287730001&run_id=297450002&run_id=314200005&run_id=306940004&run_id=312100003&run_id=314200002) ([master](https://wpt.fyi/results/workers/constructors/SharedWorker/URLMismatchError.htm?run_id=297370001&run_id=319840011&run_id=314120012&run_id=306900006&run_id=291680008&run_id=287670008&run_id=275610007&run_id=319850011))
+- [/workers/semantics/multiple-workers/004.html](https://wpt.fyi/results/workers/semantics/multiple-workers/004.html?run_id=310440006&run_id=297430002&run_id=287730001&run_id=297450002&run_id=314200005&run_id=306940004&run_id=312100003&run_id=314200002) ([master](https://wpt.fyi/results/workers/semantics/multiple-workers/004.html?run_id=297370001&run_id=319840011&run_id=314120012&run_id=306900006&run_id=291680008&run_id=287670008&run_id=275610007&run_id=319850011))
+- [/workers/shared-worker-name-via-options.html](https://wpt.fyi/results/workers/shared-worker-name-via-options.html?run_id=310440006&run_id=297430002&run_id=287730001&run_id=297450002&run_id=314200005&run_id=306940004&run_id=312100003&run_id=314200002) ([master](https://wpt.fyi/results/workers/shared-worker-name-via-options.html?run_id=297370001&run_id=319840011&run_id=314120012&run_id=306900006&run_id=291680008&run_id=287670008&run_id=275610007&run_id=319850011))
+</details>
+
+[test-analyser-minimum.html](https://wpt.fyi/results/webaudio/the-audio-api/the-analysernode-interface/test-analyser-minimum.html?run_id=297370001&run_id=319840011&run_id=314120012&run_id=306900006&run_id=291680008&run_id=287670008&run_id=275610007&run_id=319850011) is a good example of a test that could be impacted by a change.
+
+Conclusions:
+- There aren't very many single-page tests right now
+- The accidents far outnumber the intentional uses
+
+## Risks
+
+Single-page tests are intended to have as little boilerplate as possible, based on feedback from Gecko developers familiar with [Mochitest](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Mochitest), see [test example](https://github.com/mozilla/gecko-dev/blob/01c6764830acaabafeec509f5512f8ef564d6964/dom/tests/mochitest/bugs/test_protochains.html). By requiring both `single_test()` and `done()` even for sync tests where SimpleTest.js requires neither, some Gecko engineers might prefer to use Mochitest instead of WPT. However, a single sync test can also be written by just wrapping the code in `test()`. 
```

Comment:
```suggestion
Single-page tests are intended to have as little boilerplate as possible, based on feedback from Gecko developers familiar with [Mochitest](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Mochitest); see [test example](https://github.com/mozilla/gecko-dev/blob/01c6764830acaabafeec509f5512f8ef564d6964/dom/tests/mochitest/bugs/test_protochains.html). By requiring both `single_test()` and `done()` even for sync tests where SimpleTest.js requires neither, some Gecko engineers might prefer to use Mochitest instead of WPT. However, a single sync test can also be written by just wrapping the code in `test()`. 
```